src/layers/HandicapLayer/HandicapLayer.js

```diff
@@ -101,7 +133,17 @@ class HandicapLayer extends VectorLayer {
     }
 
     const isHighlighted = feature === this.clickedFeature;
-    return HandicapLayer.getIconStyle(geometry, isHighlighted);
+    const cacheKey = `${handicapType}_${isHighlighted}`;
```

Comment:
@gevos @edellucien since the geometry changes depending on the resolution, this cache will lead to stations displayed at the wrong coordinate. Not sure if a caching of styles makes sense at all.

Reply:
> since the geometry changes depending on the resolution, this cache will lead to stations displayed at the wrong coordinate.

If the feature should not be visible, the style function is left: https://github.com/geops/trafimage-maps/blob/lucien/handicap/src/layers/HandicapLayer/HandicapLayer.js#L132. And the `getIconStyle() `function isn't dependent on the resolution. So, the features should be placed correctly at the correct coordinate and resolution.

Reply:
And the `getIconStyle()` function isn't dependent on the `geometry`.